src/pages/Detail/DetailHeader/Components/DetailHeaderButtons/DetailHeaderButtons.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import './DetailHeaderButtons.scss';
+
+class DetailHeaderButtons extends React.Component {
```

Comment:
```suggestion
class DetailHeaderButtons extends Component {
```
Component import 해왔으므로 삭제 가능합니다!

Reply:
수정했습니다.

Reply:
수정했습니다.